Client/mods/deathmatch/logic/CVehicleUpgrades.cpp

```diff
@@ -64,6 +64,7 @@ bool CVehicleUpgrades::IsUpgradeCompatible(unsigned short usUpgrade)
         return false;
 
     unsigned short usModel = m_pVehicle->GetModel();
+    
```

Comment:
```suggestion
    unsigned short usModel = m_pVehicle->GetModel();
    usModel = g_pGame->GetModelInfo(usModel)->GetParentID() || usModel;
```
This do same thing, but less code.

Reply:
> This do same thing, but less code.

Your expression will product 1 all the time.